py-oxbow/pyproject.toml

```diff
@@ -24,6 +24,7 @@ dependencies = [
     "pandas>=2.2.3",
     "pyarrow>=19.0.1",
     "fsspec>=2025.5.1",
+    "typing_extensions",
```

Comment:
Does this need a python version restriction? e.g. `"typing-extensions ; python_version<'3.10'"`